.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+about: File a bug report
+title: Describe the problem
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Your issue will be triaged by the RNW team according to this process: https://github.com/microsoft/react-native-windows/wiki/Triage-Process
+-->
+## Environment
+Run the following in your terminal and copy the results here.
+1. `react-native -v`:
+2. `react-native run-windows --info`:
+3. `reg query "HKLM\SOFTWARE\Microsoft\Windows\CurrentVersion\AppModelUnlock"`
```

Comment:
Can we gather this with our `info` command instead? Why does this need to be a separate step. In the RN repo you just do `react-native info` and you're done, where we have 3 steps.

Reply:
@chrisglein  I looked into this today. The way it works is that there are 2 parts: react-native-community CLI, and envinfo. Envinfo only knowns about android and ios SDKs, editors, etc. and the community CLI only asks for these:
```js
  const info = await _envinfo().default.run({
    System: ['OS', 'CPU', 'Memory', 'Shell'],
    Binaries: ['Node', 'Yarn', 'npm', 'Watchman'],
    IDEs: ['Xcode', 'Android Studio'],
    Managers: ['CocoaPods'],
    Languages: ['Java', 'Python'],
    SDKs: ['iOS SDK', 'Android SDK'],
    npmPackages: ['react', 'react-native', '@react-native-community/cli'],
    npmGlobalPackages: ['*react-native*']
  }, options);
```

We would probably have to add the logic to envinfo to collect Windows-specific things and once that's in place, we can then add it to react-native info

Reply:
@chrisglein there is someone who was going to implement this in envinfo, so I reached out to see if they need help
https://github.com/tabrindle/envinfo/issues/155#issuecomment-661303425

Reply:
Thanks for following up and filling out that info @asklar with the [PR](https://github.com/tabrindle/envinfo/pull/178). What's the timeline for that roundtripping to affect our setup steps? I assume we take these edits and then refine again when that goes through.

Reply:
I expect the envinfo change to go in soon. After that it's a question of how soon will the CLI accept a change, and how soon after that will they cut a new version; I'm not sure about either of those but probably not immediately.

Reply:
I think these changes still have value even while we wait for envinfo (at which point we'll want to reduce the steps even further). Agree?

Reply:
agree

CLI PR here: [Add Windows info to react-native info](https://github.com/react-native-community/cli/pull/1227)